PythonExamples/mode_switch_test.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python3
+import serial
+import json
+import time
+import requests
+import argparse
+import socket
+
+class OpenIrisModeTester:
+    def __init__(self, port, baudrate=115200, timeout=5):
+        self.port = port
+        self.baudrate = baudrate
+        self.timeout = timeout
+        self.serial_conn = None
+        self.device_ip = None
+        self.device_port = None
+
+    def connect_serial(self):
+        try:
+            self.serial_conn = serial.Serial(self.port, self.baudrate, timeout=self.timeout)
+            self.flush_serial_logs()
+            print(f"Connected to {self.port} at {self.baudrate} baud")
+            return True
+        except Exception as e:
+            print(f"Error connecting to serial port: {e}")
+            return False
+
+    def flush_serial_logs(self):
+        if self.serial_conn and self.serial_conn.is_open:
+            self.serial_conn.reset_input_buffer()
+            self.serial_conn.reset_output_buffer()
+            print("Serial logs flushed.")
+
+    def send_command(self, command_obj):
+        if not self.serial_conn:
+            print("Serial connection not established")
+            return False
```

Comment:
we're returning here a False, a Json object and a None. I'd replace the `False` with a `None` just to be consistent